spec.bs

```diff
@@ -622,6 +625,33 @@ members:
 The [=payment method additional data type=] for this payment method is
 {{SecurePaymentConfirmationRequest}}.
 
+### Checking if Secure Payment Confirmation is available ### {#sctn-secure-payment-confirmation-available-api}
+
+A static API is added to {{PaymentRequest}} in order to provide developers a
+simplified method of checking whether Secure Payment Confirmation is available.
+
+<xmp class="idl">
+partial interface PaymentRequest {
+    static Promise<boolean> isSecurePaymentConfirmationAvailable();
+};
+</xmp>
+<dl dfn-type="attribute" dfn-for="PaymentRequest">
+    :   {{PaymentRequest/isSecurePaymentConfirmationAvailable()}}
+    ::  Upon invocation, a promise is returned that resolves with a value of
+        `true` if the user agent believes that the Secure Payment Confirmation
+        feature is available, or `false` otherwise.
+</dl>
+
+This allows a developer to perform the following check when deciding whether to
+initiate a SPC flow:
+
+<pre class="example" id="available-example" highlight="js">
+const spcAvailable =
+    PaymentRequest &&
+    PaymentRequest.isSecurePaymentConfirmationAvailable &&
+    await PaymentRequest.isSecurePaymentConfirmationAvailable();
+</pre>
```

Comment:
Lets add something at the bottom like the following (but maybe with better wording than me :D):

```
NOTE: The use of the static {{PaymentRequest/isSecurePaymentConfirmationAvailable}} method is recommended for
      SPC feature detection, instead of calling {{PaymentRequest/canMakePayment}} on an already-constructed
      PaymentRequest object.
```

Reply:
Done, that wording SGTM!